.github/workflows/py_performance.yml

```
@@ -0,0 +1,60 @@
+      run:
+        working-directory: python
+    strategy:
+      matrix:
+        python-version: [ 3.12 ]
+        poetry-version: [ 1.7.1 ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Debug workflow trigger
+        run: |
```

actionlint (1.7.7)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)